moviepy/Clip.py

```diff
@@ -7,8 +7,17 @@
 from numbers import Real
 from operator import add
 
-import numpy as np
-import proglog
+try:
+    import numpy as np
+except ImportError as exc:
+    raise ImportError("MoviePy requires numpy. Please install it with pip install numpy") from exc
```

Comment:
What is the benefit of doing this? `numpy`, and other dependencies are clearly stated in the `pyproject.toml` file so I think adding this is overkill.

Maybe I am missing some other benefit?